delfin/drivers/dell_emc/vmax/alert_handler.py

```diff
@@ -17,129 +17,130 @@
 from oslo_log import log
 
 from delfin import exception
+from delfin.alert_manager import alert_processor
 from delfin.drivers.dell_emc.vmax import alert_mapper
 
 LOG = log.getLogger(__name__)
 
 
 class AlertHandler(object):
     """Alert handling functions for vmax driver"""
-    default_category = 'New'
-    default_event_type = 'unknown'
-    default_probable_cause = 'Not available'
-    default_severity = 'unknown'
-    default_me_category = 'storage-subsystem'
-    default_location = 'unknown'
 
-    # Mandatory attributes expected in alert info to proceed with model filling
-    mandatory_alert_attributes = ('storage_id', 'storage_name', 'vendor',
-                                  'model', 'connUnitName', 'emcAsyncEventCode')
+    # Translation of trap severity to alert model severity
+    SEVERITY_MAP = {"emergency": alert_processor.Severity.FATAL,
+                    "alert": alert_processor.Severity.CRITICAL,
+                    "critical": alert_processor.Severity.CRITICAL,
+                    "error": alert_processor.Severity.MAJOR,
+                    "warning": alert_processor.Severity.WARNING,
+                    "notify": alert_processor.Severity.WARNING,
+                    "info": alert_processor.Severity.INFORMATIONAL,
+                    "debug": alert_processor.Severity.INFORMATIONAL,
+                    "mark": alert_processor.Severity.INFORMATIONAL}
+
+    # Translation of trap resource type to alert model resource type
+    RESOURCE_TYPE_MAP = {
```

Comment:
why need resource_type_map? vmax is a storage array

Reply:
This is as per their mib,alert can contain any of these.  In our case, Alert might be mainly related to storage which will get converted to STORAGE